src/style/search/search-bar.less

```diff
@@ -50,7 +50,7 @@
     left: 0;
     right: 0;
     bottom: 0;
-    background-color: rgba(0,0,0,0);
+    background-color: rgba(0,0,0,0.01);
```

Comment:
Honestly I'm all for increasing the transparency so we make it clear to the user that they're in a modal state. With huge monitors, it's easy to miss the search bar in the middle while looking at the panels in the right.